### PR TITLE
Merge release 2021.01.29 into main

### PR DESCRIPTION
**Pull request by jameslamb**
Replaces https://github.com/saturncloud/examples/pull/87, to fix conflicts. This merges recent changes from 2021.01.29 into `main`.

### CODE REVIEW

File: examples/examples-gpu/nyc-taxi-snowflake/rf-scikit.ipynb [hunk 1]
```diff
@@ -403,6 +407,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "query_result = conn.cursor().execute(query, \"2019-02-01\")\n",
```
Comment (jameslamb) [inline examples/examples-gpu/nyc-taxi-snowflake/rf-scikit.ipynb]:
```suggestion
```

**Comment by jameslamb** [comment]
@rikturr can you please take a look and let me know if you have any questions?

**Review by rikturr — APPROVED** [review]

File: examples/examples-gpu/nyc-taxi-snowflake/rf-scikit.ipynb [hunk 2]
```diff
@@ -190,7 +190,11 @@
     "features = numeric_feat + categorical_feat\n",
     "y_col = \"high_tip\"\n",
     "\n",
-    "taxi_train = taxi[features + [y_col]].astype(float).fillna(-1)"
+    "query_result = conn.cursor().execute(query, \"2019-01-01\")\n",
+    "columns = [x[0] for x in query_result.description]\n",
+    "taxi_train = pd.DataFrame(query_result.fetchall(), columns=columns)\n",
+    "taxi_train.columns = taxi_train.columns.str.lower()"
```
Comment (jameslamb) [inline examples/examples-gpu/nyc-taxi-snowflake/rf-scikit.ipynb]:
```suggestion
    "taxi_train.columns = taxi_train.columns.str.lower()",
```

Comment (jameslamb) [inline examples/examples-gpu/nyc-taxi-snowflake/rf-scikit.ipynb]:
```suggestion
    "taxi_train.columns = taxi_train.columns.str.lower()",
```

**Comment by jameslamb** [comment]
Will fix and merge this in a bit. Now that some changes had to be made to resolve conflicts, I need to spin up `examples-gpu` on app.internal and test again.

**Pull request merged**